### PR TITLE
Collect more iBridge data

### PR DESCRIPTION
**Pull request by tuxudo**
Collects more iBridge and machine information. Contains more widgets, database migration, and new client script

### CODE REVIEW

**Comment by bochoven** [comment]
LGTM do you want more 👀 on this?

**Comment by tuxudo** [comment]
It's a bigger PR, mostly wanting your review on it before it's merged.

File: ibridge_controller.php [hunk 1]
```diff
@@ -8,62 +8,54 @@
...
-     public function get_ibridge()
-     {
-        $obj = new View();
+    public function get_scroll_widget($column)
+    {
+        $sql = "SELECT COUNT(CASE WHEN ".$column." <> '' AND ".$column." IS NOT NULL THEN 1 END) AS count, ".$column." 
+                FROM ibridge
+                LEFT JOIN reportdata USING (serial_number)
+                ".get_machine_group_filter()."
+                AND ".$column." <> '' AND ".$column." IS NOT NULL
```
Comment (bochoven) [inline ibridge_controller.php:43]: Inserting $column in an $sql query like this is a SQL injection vulnerability. Check https://github.com/munkireport/softwareupdate/blob/master/softwareupdate_controller.php#L53 for a safer method. We might even put that in the base controller

Comment (tuxudo) [reply ibridge_controller.php:43]: Fixed in latest PR with alphanumeric filtering

**Pull request merged**